.github/workflows/moodle-ci.yml

```diff
@@ -20,27 +20,24 @@ jobs:
         ports:
           - 5432:5432
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 3
+
       mariadb:
         image: mariadb:10
         env:
           MYSQL_USER: 'root'
           MYSQL_ALLOW_EMPTY_PASSWORD: "true"
           MYSQL_CHARACTER_SET_SERVER: "utf8mb4"
           MYSQL_COLLATION_SERVER: "utf8mb4_unicode_ci"
-
         ports:
           - 3306:3306
         options: --health-cmd="mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries 3
 
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.4', '8.0']
-        moodle-branch: ['MOODLE_400_STABLE', 'MOODLE_401_STABLE', 'master']
+        php: ['7.4', '8.0', '8.1']
+        moodle-branch: ['MOODLE_401_STABLE']
```

Comment:
Reicht hier die eine Version?

Reply:
Alles klar! Dass der Support im Dezember endet wusste ich nicht.

Die CI Checks failen weil `master` php 8.0 bis 8.1 voraussetzt oder? Vielleicht kann man das mit einer komplexeren [Matrix Konfiguration](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) lösen.

Reply:
Einen entsprechenden Exclude hatten wir auch vorher drin, habe ich wieder hinzugefügt